analyses/infercnv-consensus-cell-type/scripts/01b_run-infercnv-internal-reference.R

```diff
@@ -0,0 +1,305 @@
+#!/usr/bin/env Rscript
```

Comment:
I think what I would do instead is modify the `run-infercnv.R` script that you already have to do the following: 

- Make the option for `reference_file` optional. 
- Add two new options for `reference_celltype_group` and `celltype_file` that are optional. 
- If no `reference_file` is provided, then you need to provide a `reference_celltype_group` and `celltype_file`.
- Handle creating the `raw_counts_matrix` and annotations file with an if statement depending on using a pooled or internal reference.